src/app/handlers/auto_unverify.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { IContainer, IHandler, IMessage } from '../../common/types';
+
+export class AutoUnverifyHandler implements IHandler {
+  constructor(public container: IContainer) {}
+
+  public async execute(message: IMessage): Promise<void> {
+    const avatar = message.author.avatar; //This is null if they have default avatar
+
+    if (!avatar) {
```

Comment:
nothing is necessarily wrong here, but a good pattern u should adopt is the use of guard clauses.

more spceifically, a guard clause basically refers to having guards against your code with early returns at the top, and at the bottom of the code is your "happy" case.

for this, it would be something like

```
if hasAvatar
  return;

// execute the code since the happy case here is not having an avatar
```

it's not that obvious the benefits of using a guard clause here, but its something that will come up again and it will make your code way more maintainable.

Reply:
Yeah, I've been learning that from Corgo, didn't think ab it here cuz the "Happy Case" wasn't large. Will change